cfme/containers/provider/__init__.py

```diff
@@ -1,16 +1,14 @@
-import attr
-
 import random
+import re
 from random import sample
-from six import string_types
 from traceback import format_exc
 
-
-import re
+import attr
 from navmazing import NavigateToSibling, NavigateToAttribute
+from six import string_types
 from widgetastic_manageiq import StatusBox, ContainerSummaryTable
```

Comment:
Required : Line 9 and 11 can be merged.

from widgetastic_manageiq import Accordion, ContainerSummaryTable, LineChart, ManageIQTree, StatusBox, SummaryTable